src/features/events_list/components/FullState/FullState.tsx

```diff
@@ -53,6 +53,17 @@ export function FullState({
     }
   }, [currentEventId, eventsList, virtuoso, setHasUnlistedEvent]);
 
+  const currentEventRef = useRef(currentEventId);
+  currentEventRef.current = currentEventId;
+  const eventClickHandler = useCallback(
+    (id: string) => {
+      if (id !== currentEventRef.current) {
+        onCurrentChange(id);
+      }
+    },
+    [onCurrentChange, currentEventRef],
```

Comment:
Do we need to include it in deps array? 

`currentEventRef` will be immutable

Reply:
actually is no difference are we include refs in deops array or not, but linter more happy when we add =)